tests/Backend/Workspaces/WorkspacesTestCase.php

```diff
@@ -25,7 +25,9 @@ private function deleteAllWorkspaces()
     {
         $workspaces = new Workspaces($this->_client);
         foreach ($workspaces->listWorkspaces() as $workspace) {
-            $workspaces->deleteWorkspace($workspace['id']);
+            $workspaces->deleteWorkspace($workspace['id'], [
+                'async' => true,
```

Comment:
Proč?

Reply:
Protože jsou ty test pak nestabilní a random padají. Protože tím nahrneš velkou dávku dotazů přímo na DB backend. Popsal jsem to myslím i v Jiře.

Reply:
To je přece úplně naopak, ne? Když tady budu v cyklu volat DB dotazy, tak jich najednou bude méně, než když ty samé dotazy budou zpracovávat paralelně třeba 4 workery, ne?

Reply:
Už, zapomněl jsem, že to PHP volání čeká. Ale rozdíl stejně moc nechápu.

Reply:
Jakože 

> Pomoc by mohlo když by se místo sync DELETE volal async DELETE https://keboola.docs.apiary.io/#reference/workspaces/manage-workspace/delete-workspace

Mi nepřijde, že odpovídá na to, proč by to mělo pomoct. Jakože, stejně by teda asi fungovalo třeba 100ms sleep za tím smazáním, ne?

Reply:
No ale ten cyklus nevoláš v jednu chvíli z jednoho místa. Ale v tuhle chvíli z N suite. Tím že to jde přes workery tak tu zátěž rozložíš. A pomoct by to mělo, protože od doby co jsem to tak udělal tak testy procházely.
Dělat alchymii se sleepama nebudu, to bude ve stavu v jakém to bylo v létě.

Reply:
> No ale ten cyklus nevoláš v jednu chvíli z jednoho místa. Ale v tuhle chvíli z N suite. 

Jasně, to je ta úvaha, co mi nedošla. Díky.